shell/platform/windows/compositor_opengl_unittests.cc

```diff
@@ -185,53 +182,7 @@ TEST_F(CompositorOpenGLTest, PresentEmpty) {
   EXPECT_CALL(*surface(), IsValid).WillRepeatedly(Return(true));
   EXPECT_CALL(*surface(), MakeCurrent).WillOnce(Return(true));
   EXPECT_CALL(*surface(), SwapBuffers).WillOnce(Return(true));
-  EXPECT_TRUE(compositor.Present(view()->view_id(), nullptr, 0));
-}
-
-TEST_F(CompositorOpenGLTest, HeadlessPresentIgnored) {
```

Comment:
The following tests were all replaced by `WindowsTest.PresentHeadless`:

1. `CompositorOpenGLTest.HeadlessPresentIgnored`
2. `CompositorOpenGLTest.UnknownViewIgnored`
3. `CompositorSoftwareTest.HeadlessPresentIgnored`
4. `CompositorSoftwareTest.UnknownViewIgnored`

(Presenting to the implicit view while in headless mode is effectively the same thing as presenting to an unknown view)